packages/legend-application-studio/src/components/editor/edit-panel/connection-editor/post-processor-editor/MapperPostProcessorEditor.tsx

```diff
@@ -0,0 +1,283 @@
+        ).length >= 2;
+
+    const isMapperDuplicated = (val: Mapper): boolean =>
+      isSchemaMapperDuplicated(val) || isTableMapperDuplicated(val);
+
+    return (
+      <ResizablePanelGroup orientation="horizontal">
+        <ResizablePanel>
+          <ResizablePanelGroup orientation="vertical">
+            <ResizablePanel size={170} minSize={70}>
+              <div className="panel__container panel__container--border-right">
+                <DropdownMenu
+                  content={
+                    <MenuContent>
+                      <MenuContentItem onClick={addSchemaMapper}>
+                        New Schema Mapper
+                      </MenuContentItem>
+                      <MenuContentItem onClick={addTableMapper}>
+                        New Table Mapper
+                      </MenuContentItem>
+                    </MenuContent>
+                  }
+                  menuProps={{
+                    anchorOrigin: { vertical: 'bottom', horizontal: 'right' },
+                    transformOrigin: { vertical: 'top', horizontal: 'right' },
+                    elevation: 7,
+                  }}
+                >
+                  <PanelHeader title="mapper">
+                    <PanelHeaderActionItem tip="Create Mapper">
+                      <PlusIcon />
+                    </PanelHeaderActionItem>
+                  </PanelHeader>
+                </DropdownMenu>
+
+                <div className="panel__content">
+                  {mappers.map((mapper) => (
+                    <ContextMenu
+                      key={mapper._UUID}
+                      disabled={false}
+                      content={
+                        <MenuContent>
+                          <MenuContentItem onClick={deleteMapper(mapper)}>
+                            Delete
+                          </MenuContentItem>
+                        </MenuContent>
+                      }
+                      menuProps={{ elevation: 7 }}
+                    >
+                      <PanelExplorerItem
+                        title={
+                          mapper instanceof TableNameMapper
+                            ? 'Table Mapper'
+                            : 'Schema Mapper'
+                        }
+                        validationErrorMessage={
+                          isMapperDuplicated(mapper)
+                            ? 'Mappers have the same values'
+                            : undefined
+                        }
+                        isSelected={
+                          mapper === postProcessorState.selectedMapper
+                        }
+                        onSelect={() => selectMapper(mapper)}
+                      />
+                    </ContextMenu>
+                  ))}
+                </div>
+              </div>
+            </ResizablePanel>
+
+            <ResizablePanelSplitter />
+
+            <ResizablePanel>
+              <div className="panel__container">
```

Comment:
I think this is now how we want this. We have `panel` which contains `panel__header` and `panel__content`. I don't see your `panel__content` here. And I don't see `panel` neither. Let's create `Panel` and `PanelContent` component